CSharpMath/Atoms/Factories/MathAtoms.cs

```diff
@@ -156,81 +156,81 @@ public static class MathAtoms {
-         { "varointclockwise", Operator("\u2232" , false) }, //not in iosMath
-         { "ointctrclockwise", Operator("\u2233" , false) }, //not in iosMath
-         { "bigwedge", Operator("\u22C0" , null) },
-         { "bigvee", Operator("\u22C1" , null) },
-         { "bigcap", Operator("\u22C2" , null) },
-         { "bigcup", Operator("\u22C3" , null) },
-         { "bigbot", Operator("\u27D8" , null) }, //not in iosMath
-         { "bigtop", Operator("\u27D9" , null) }, //not in iosMath
-         { "bigodot", Operator("\u2A00" , null) },
-         { "bigoplus", Operator("\u2A01" , null) },
-         { "bigotimes", Operator("\u2A02" , null) },
-         { "bigcupdot", Operator("\u2A03" , null) }, //not in iosMath
-         { "biguplus", Operator("\u2A04" , null) },
-         { "bigsqcap", Operator("\u2A05" , null) }, //not in iosMath
-         { "bigsqcup", Operator("\u2A06" , null) },
-         { "bigtimes", Operator("\u2A09" , null) }, //not in iosMath
+         { "prod", Operator("prod","\u220F" , null) },
+         { "coprod", Operator("coprod","\u2210" , null) },
+         { "sum", Operator("sum","\u2211" , null) },
+         { "int", Operator("int","\u222B" , false) },
+         { "iint", Operator("iint","\u222C", false) }, //not in iosMath
+         { "iiint", Operator("iiint","\u222D", false) }, //not in iosMath
+         { "iiiint", Operator("iiiint","\u2A0C", false) }, //not in iosMath
+         { "oint", Operator("oint","\u222E" , false) },
+         { "oiint", Operator("oiint","\u222F" , false) }, //not in iosMath
+         { "oiiint", Operator("oiiint","\u2230" , false) }, //not in iosMath
+         { "intclockwise", Operator("intclockwise","\u2231" , false) }, //not in iosMath
+         { "awint", Operator("awint","\u2A11" , false) }, //not in iosMath
+         { "varointclockwise", Operator("varointclockwise","\u2232" , false) }, //not in iosMath
+         { "ointctrclockwise", Operator("ointctrclockwise","\u2233" , false) }, //not in iosMath
+         { "bigwedge", Operator("bigwedge","\u22C0" , null) },
+         { "bigvee", Operator("bigvee","\u22C1" , null) },
+         { "bigcap", Operator("bigcap","\u22C2" , null) },
+         { "bigcup", Operator("bigcup","\u22C3" , null) },
+         { "bigbot", Operator("bigbot","\u27D8" , null) }, //not in iosMath
+         { "bigtop", Operator("bigtop","\u27D9" , null) }, //not in iosMath
+         { "bigodot", Operator("bigodot","\u2A00" , null) },
+         { "bigoplus", Operator("bigoplus","\u2A01" , null) },
+         { "bigotimes", Operator("bigotimes","\u2A02" , null) },
+         { "bigcupdot", Operator("bigcupdot","\u2A03" , null) }, //not in iosMath
+         { "biguplus", Operator("biguplus","\u2A04" , null) },
+         { "bigsqcap", Operator("bigsqcap","\u2A05" , null) }, //not in iosMath
+         { "bigsqcup", Operator("bigsqcup","\u2A06" , null) },
+         { "bigtimes", Operator("bigtimes","\u2A09" ,  null) }, //not in iosMath

```

Comment:
Formatting of spaces

Reply:
I've been using regex from visual studio for batch refactoring, thats why i have problems with spaces, but it easy to fix.